### DISCUSSION TITLE
Config settings with en_core_web_md to train a NER

### DISCUSSION THREAD

**Discussion by oliviercwa** [explosion/spaCy#9098] [Help: Coding & Implementations] [answered]
Hello,
I am trying to use train a NER using the en_core_wb_md model. 
The config generator proposes configuration for both en_core_web_sm and en_core_web_lg but not en_core_web_md.

I have started from the en_core_web_lg config and modified this
```
[initialize]
vectors = "en_core_web_md"
```

I was wondering if other sections should be modified, especially these two
```
[components.tok2vec.model.embed]
@architectures = "spacy.MultiHashEmbed.v2"
width = ${components.tok2vec.model.encode.width}
attrs = ["ORTH","SHAPE"]
rows = [5000,2500]
include_static_vectors = true

[components.tok2vec.model.encode]
@architectures = "spacy.MaxoutWindowEncoder.v2"
width = 256
depth = 8
window_size = 1
maxout_pieces = 3
```

Thank you

**Comment by svlandeg** [comment, marked as answer]
Hi!

You should be able to just change `en_core_web_lg` to `en_core_web_md` everywhere and then it should work fine, as both pretrained pipelines are the same in architecture.

Let us know if you run into any issues though.

**Reply by oliviercwa** [reply]
Hi,
It works fine however on the exact same dataset (about 5000 records), using the default config (spacy 3.1.2), the F1 score after training is about 20 points lowered compared to training using spacy 2.1.9 with en_core_web_md model. 

Is this expected ?

**Reply by svlandeg** [reply]
That feels like a very large gap. Perhaps the training stopped earlier? How many epochs did you train for in both cases?

**Reply by oliviercwa** [reply]
was about 200 iteration in the first case and 15 epochs in the second case. I can try to train for longer but the model seems "stuck" and does not appear to progress anymore. The Loss NER is also stable and does not reduce anymore like with spacy 2.1.9

**Reply by svlandeg** [reply]
How exactly did you define your model for v2? Did you use only the vectors, or did you start from the fully pretrained NER component?

**Reply by oliviercwa** [reply]
We only used the vocab and vector from the en_core_web_md model, then the NER was trained from scratch
